File: src/main/java/org/jukeboxmc/console/ConsoleSender.java
package org.jukeboxmc.console;

import org.jukeboxmc.Server;
import org.jukeboxmc.command.CommandSender;

/**
 * @author LucGamesYT
 * @version 1.0
 */
public class ConsoleSender implements CommandSender {

    private final Server server;

    public ConsoleSender( Server server ) {
        this.server = server;
    }

    @Override
    public void sendMessage( String message ) {
        this.server.getLogger().info( message );
    }

    @Override
    public boolean hasPermission( String permission ) {
        return true;
    }

    @Override
    public Server getServer() {
        return this.server;
    }
}

File: JukeboxMC-API/src/main/kotlin/org/jukeboxmc/api/command/ConsoleSender.kt
package org.jukeboxmc.api.command

import org.jukeboxmc.api.JukeboxMC
import org.jukeboxmc.api.Server

class ConsoleSender : CommandSender {

    override fun sendMessage(message: String) {
        JukeboxMC.getServer().getLogger().info(message)
    }

    override fun hasPermission(permission: String): Boolean {
        return true
    }

    override fun getServer(): Server {
        return JukeboxMC.getServer()
    }
}